selectrionsort.py

```diff
@@ -0,0 +1,15 @@
+print("Enter number of elements")
+N=int(input())

```

Comment:
Never assign a global/common variable (which is used across) in a single letter, make it understandable/readable, like ``` userInput```